### ISSUE TITLE
Log test results to Tensorboard during training?

### ISSUE THREAD

**Issue by rumozam** [open-mmlab/mmsegmentation#2211]
Hi there! Thank you for putting so much time and effort into mmsegmentation!
Is there an option that makes it possible to evaluate not only on the validation set but also on the test set during training, and then log it, such that it can be displayed in Tensorboard?
I.e. I would like to have not only the train and val results on Tensorboard, but also the test results, because Tensorboard makes a really nice graphic with the results for each single class, which is great for trouble-shooting.

Alternatively, is there a way to test all saved checkpoints in one go and log the results to Tensorboard? I found only scripts for testing a single checkpoint.

Thank you again!

**Comment by MeowZheng** [comment]
You can try to revise the test/val dataset config and make it same as train dataset.

If you would like to use MMSeg1.0.0rc0 with Tensorboard, please refer to the visualization.md, https://github.com/open-mmlab/mmsegmentation/blob/dev-1.x/docs/en/user_guides/visualization.md

**Comment by rumozam** [comment]
Thank you! I will try to add a new EvalHook for the test data.
